x-pack/platform/plugins/private/upgrade_assistant/server/lib/reindexing/reindex_service.ts

```diff
@@ -169,7 +169,14 @@ export const reindexServiceFactory = (
    * @param reindexOp
    */
   const setReadonly = async (reindexOp: ReindexSavedObject) => {
-    const { indexName } = reindexOp.attributes;
+    const { indexName, rollupJob } = reindexOp.attributes;
+
+    if (rollupJob) {
+      // will wait up to 30 seconds for the job to stop.
```

Comment:
I've reached out to get an answer on this but I'm also prepared to move forward with it as it